crates/uv-auth/src/middleware.rs

```diff
@@ -374,6 +384,7 @@ impl AuthMiddleware {
         extensions: &mut Extensions,
         next: Next<'_>,
         url: &str,
+        maybe_index_url: Option<&Url>,
```

Comment:
Personally, I'd just call this `index_url`, since `maybe` is implied by the type.